src/pyhf/workspace.py

```diff
@@ -362,74 +604,34 @@ def combine(cls, left, right):
         Args:
             left (~pyhf.workspace.Workspace): A workspace
             right (~pyhf.workspace.Workspace): Another workspace
+            join (:obj:`str`): How to join the two workspaces. Pick from "none", "outer", "left outer", or "right outer".
```

Comment:
> Pick from "none", "outer", "left outer", or "right outer".

@kratsg Do you think that we should explain somewhere that this is using SQL syntax? Or link to some documentation on what this means? I can see that we might be able to say "yo, if you're joining then you should know what you're doing" but I'm thinking about the uninitiated grad student that has been told to go do something.

Reply:
dunno. i'm kinda pessimistic and go with "nah" but...

Reply:
can't we just let this be called `prefer_left` and `prefer_right`?

Reply:
I mean, this is a very _common_ concept: https://en.wikipedia.org/wiki/Join_(SQL) -- so I'd rather stick with the definition of it. We can add an additional option to feed in "prefer left outer" and "prefer right outer" (because there are also inner joins) and those translate to left outer / right outer.